idunn/api/places_list.py

```diff
@@ -1,31 +1,52 @@
+import os
 import logging
 from elasticsearch import Elasticsearch
 from apistar.exceptions import BadRequest
 
-from idunn.utils.settings import Settings
+from idunn.utils.settings import Settings, _load_yaml_file
 from idunn.utils.index_names import IndexNames
 from idunn.places import POI
 from idunn.api.utils import fetch_bbox_places, LONG, SHORT, DEFAULT_VERBOSITY_LIST
 
 from apistar import http
 
 from pydantic import BaseModel, ValidationError, validator
-from typing import List, Optional
+from pydantic.error_wrappers import ErrorWrapper
+from typing import List, Optional, Any
 
 logger = logging.getLogger(__name__)
 
 VERBOSITY_LEVELS = [LONG, SHORT]
 
-MAX_WIDTH = 1.0 # max bbox longitude in degrees
+MAX_WIDTH = 1.0 # max bbox longitude in degrees
 MAX_HEIGHT = 1.0  # max bbox latitude in degrees
 
+
+class Categories:
+    _categories = None
+
+    @classmethod
+    def get_categories(cls):
+        if cls._categories is None:
+            categories_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "../utils/categories.yml")
+            if categories_path:
+                categories = _load_yaml_file(categories_path)
+        return categories.get('categories')
```

Comment:
this is too complicated, and `_categories` is not used.
Why not removing this class, and simply use:
```python
def get_categories():
   categories_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "../utils/categories.yml")
   return _load_yaml_file(categories_path)['categories']

all_categories = get_categories()
```
so that the yaml file is read only once.